### PR TITLE
Added support for auth schemes and Release v0.1.5

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,54 +34,72 @@ A grouping element to hold other elements.
...
 ```js
 let groups = category.transitionGroups;
 ```
 
+##### category.authSchemes
+Get an array element of all child element with element name equal to one of the names given below. This property is **read-only**.
+
+* Basic Authentication Scheme
+* Token Authentication Scheme
+* Oauth2 Scheme
```
Comment (kylef) [inline README.md]: I know this is how it is in the Refract spec, but I think we should start moving to "OAuth2" spelling as per the OAuth spec before we end up having it "Oauth2" everywhere and can't change.

File: README.md [hunk 2]
```diff
@@ -34,54 +34,72 @@ A grouping element to hold other elements.
...
+
+```js
+let schemes = category.authSchemes;
+```
+
+##### category.authSchemeGroupss
```
Comment (kylef) [inline README.md]: `Groupss` is the double S correct?

File: README.md (hunk 1 above)
Comment (pksunkara) [inline README.md]: I was just thinking about it today. @smizell What do you think?

File: README.md (hunk 2 above)
Comment (pksunkara) [inline README.md]: Will fix.

**Comment by kylef** [comment]
👍 other than my two minor comments.

File: README.md (hunk 1 above)
Comment (smizell) [inline README.md]: I think we should do it. I think it's important to have right and seems like an easy fix now.

**Comment by pksunkara** [comment]
@kylef Updated.

**Pull request merged**